meta-balena-common/recipes-bsp/grub/grub-conf.bb

```diff
@@ -0,0 +1,41 @@
+SUMMARY = "Grub configuration and other various files"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
+    file://grub.cfg_external \
+    file://grub.cfg_internal_template \
+    file://grubenv \
+    "
+
+inherit deploy nopackages
+
+INHIBIT_DEFAULT_DEPS = "1"
+BOOTLOADER_TIMEOUT ?= "3"
```

Comment:
In symmetry with U-Boot, we don't want any OS variant (prod/dev) to be able to enter the grub shell. In the U-Boot case we allow to overwrite this with the `OS_DEV_UBOOT_DELAY` variable. it should be the same for grub.